Blockzilla/UIConstants.swift

```diff
@@ -219,7 +223,7 @@ struct UIConstants {
         static let CardTextSearch = NSLocalizedString("Intro.Slides.Search.Description", tableName: "Intro", value: "Searching for something different? Choose a different search engine.", comment: "Description for the 'Favorite Search Engine' panel in the First Run tour.")
         static let AddSearchEngineButton = NSLocalizedString("Settings.Search.AddSearchEngineButton", value: "+ Add Another Search Engine", comment: "Text for button to add another search engine in settings")
         static let AddSearchEngineTitle = NSLocalizedString("Settings.Search.AddSearchEngineTitle", value: "Add Search Engine", comment: "Title on add search engine settings screen")
-        static let Save = NSLocalizedString("Save", value: "Save", comment: "Save button label")
+        static let save = NSLocalizedString("Save", value: "Save", comment: "Save button label")
```

Comment:
I ran into the case inconsistency hear. Should the string variables start with an uppercase or lowercase?

Reply:
Thats a good question. It looked like everything started with lowercase, but we lost some strictness along the way. I'll have to go in and fix it later.